common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -240,6 +240,15 @@
   "feature.wynntils.lootrunUtils.undoNotFarEnough": "You haven't moved far enough away to undo movement.",
   "feature.wynntils.lootrunUtils.undoStandNear": "Stand near a previous spot to undo movements.",
   "feature.wynntils.lootrunUtils.undoSuccessful": "Successfully undid movements.",
+  "feature.wynntils.miniMapOverlay.name": "Mini Map Overlay",
+  "feature.wynntils.miniMapOverlay.overlay.miniMap.followPlayerRotation.description": "Should the minimap be locked facing north or rotate based on the direction you're facing?",
+  "feature.wynntils.miniMapOverlay.overlay.miniMap.followPlayerRotation.name": "Follow Player Rotation",
+  "feature.wynntils.miniMapOverlay.overlay.miniMap.maskType.description": "Should the minimap be a rectangle or a circle? (Circle is unsupported as of now)?",
+  "feature.wynntils.miniMapOverlay.overlay.miniMap.maskType.name": "Map Mask Type",
+  "feature.wynntils.miniMapOverlay.overlay.miniMap.renderUsingLinear.description": "Should the map be rendered using linear textures to avoid aliasing issues?",
+  "feature.wynntils.miniMapOverlay.overlay.miniMap.renderUsingLinear.name": "Use Linear Rendering",
+  "feature.wynntils.miniMapOverlay.overlay.miniMap.scale.description": "How zoomed in should the map be?",
+  "feature.wynntils.miniMapOverlay.overlay.miniMap.scale.name": "Map Scale",
```

Comment:
`feature.wynntils.miniMapOverlay.overlay.miniMap.name` and `...description` is missing, so it looks odd in the Overlay Manager.

Reply:
Will do